Feature/add main

PavloGPS opened the pull request.

PavloGPS committed:
Update index.html

1. change page title.
2. add inclusion of main.html.

PavloGPS committed:
Create main.html

PavloGPS committed:
Create hero.html

PavloGPS committed:
Create details-of-our-restaurant.html

PavloGPS committed:
Create details-of-our-hotel.html

PavloGPS committed:
Create about-our-restaurant.html

PavloGPS committed:
Create about-our-hotel.html

AntOn2415 commented on src/index.html:
```diff
@@ -3,13 +3,14 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Page title</title>
+    <title>Main page</title>
```
чому назва страниці саме "Main"?

PavloGPS replied:
Структура взята з ТЗ

AntOn2415 commented on src/index.html:
```diff
@@ -3,13 +3,14 @@
     <!-- ❗️❗️❗️ Add partials like so for every page -->
     <include src="./partials/example.html"></include>
     <include src="./partials/header.html"></include>
+    <include src="./partials/main.html"></include>
```
main.html не потрібен. Можливо Краще всю основну структуру розмістити у index.html закритим тегом...

PavloGPS replied:
ок. зараз виправлю

AntOn2415 reviewed (commented):
Глянув.

PavloGPS committed:
Update index.html

1.removes main.html
2. adds section <main>
3. inside of section main adds inclusion of partials: her,about-our-restaurant...

PavloGPS committed:
new partials
